modules/logging-rn-5.5.16.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+// cluster-logging-release-notes.adoc
+// logging-5-5-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-16_{context}"]
+= Logging 5.5.16
+This release includes link:https://access.redhat.com/errata/RHSA-2023:5096[OpenShift Logging Bug Fix Release 5.5.16].
+
+[id="openshift-logging-5-5-16-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, the LokiStack gateway cached authorized requests very broadly. As a result, this caused wrong authorization results. With this update, LokiStack gateway caches on a more fine-grained basis which resolves this issue. (link:https://issues.redhat.com/browse/LOG-4434[LOG-4434])
+
```

Comment:
```suggestion
* Previously, the LokiStack gateway cached authorized requests very extensively. This resulted in incorrect authorization results. With this update, the LokiStack gateway performs finer-grained caching, which resolves this issue. (link:https://issues.redhat.com/browse/LOG-4434[LOG-4434])

```

Reply:
@agantony According to the [release notes section in the supp style guide](https://redhat-documentation.github.io/supplementary-style-guide/#release-notes), the comments above are in compliance. E.g. Using "Before this update" to start the sentence.

Reply:
Also, I would add that the suggeated change of the wording to `This resulted in` goes against [IBM SG guidance](https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references) to "Ensure that the noun to which a pronoun refers is clear". In the suggested alternative, `This` has an unclear reference -- "this what?".

Reply:
@agantony For all of Logging RNs bug fixes, we have constantly followed the "Before this update <bug description>" and "With this update <fix description>" format, so I have not made any changes to the Release Notes text. Please give your lgtm if there's no more changes/suggestions. Thank you! :)